src/vs/server/node/webClientServer.ts

```diff
@@ -106,9 +106,9 @@ export class WebClientServer {
 		@IProductService private readonly _productService: IProductService,
 	) {
 		this._webExtensionResourceUrlTemplate = this._productService.extensionsGallery?.resourceUrlTemplate ? URI.parse(this._productService.extensionsGallery.resourceUrlTemplate) : undefined;
-		const qualityAndCommit = `${_productService.quality ?? 'oss'}-${_productService.commit ?? 'dev'}`;
-		this._staticRoute = `/${qualityAndCommit}/static`;
-		this._callbackRoute = `/${qualityAndCommit}/callback`;
+		const serverRootPath = getServerRootPath(_productService);
```

Comment:
Probably the intent here was to use `getRemoteServerRootPath` from `vs/platform/remote/common/remoteHosts`